apis/dataprotection/v1alpha1/backuppolicy_types.go

```diff
@@ -112,10 +118,10 @@ type CommonBackupPolicy struct {
 
 type PersistentVolumeClaim struct {
 	// the name of PersistentVolumeClaim to store backup data.
-	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MaxLength=63
-	// +kubebuilder:default=kb-backup-data
-	Name string `json:"name"`
+	// +kubebuilder:validation:Pattern:=`^[a-z0-9]([a-z0-9\.\-]*[a-z0-9])?$`
```

Comment:
the regular expression validation here will cause incompatibility with older versions, and will report an error when old version's `Name` is an empty string.

Reply:
Done.

Got it. I thought changing to a pointer would be enough, but I didn’t consider the existing objects.